src/main/java/com/oop/lesson8/CanPlaySound.java

```diff
@@ -0,0 +1,5 @@
+package com.oop.lesson8;
+
+public abstract interface CanPlaySound {
+    public void playSound();
```

Comment:
можна не писати, та і для метода я б не рекомендував але це чисто моя особиста думка
```suggestion
public interface CanPlaySound {
   void playSound();
```